kustomize/Makefile

```diff
@@ -0,0 +1,3 @@
+export KUSTOMIZE_ROOT=$(shell cd .. && pwd)
```

Comment:
This is the entire reason this one isn't a symlink. Since the repo root has "kustomize" in it at least once (and e.g. CI has it _twice_), it was too hard to develop a regex for the generic modules Makefile that would handle this module in particular.